NuRadioReco/utilities/data/galactic_calibration/LOFAR_LBA_galactic_30_80.txt

```diff
@@ -0,0 +1,102 @@
+# Galactic calibration curve for LOFAR (30-80 MHz)
```

Comment:
I hope there is a README for this file somewhere ...

Reply:
I have added a little bit more information and also a reference to the stationGalacticCalibration.py file, where it is explained more in depth what these files contain